packages/swapkit/helpers/src/modules/swapKitError.ts

```diff
@@ -22,6 +22,8 @@ const errorCodes = {
   core_wallet_keplr_not_installed: 10107,
   core_wallet_okx_not_installed: 10108,
   core_wallet_keepkey_not_installed: 10109,
+  core_wallet_talisman_not_installed: 10110,
+  core_wallet_not_keypair_wallet: 10111,
```

Comment:
What wallet is this referring to?

Reply:
This error is used to identify the keypair wallet.
The talisman and polkadot.js is not a keypair wallet.